CHANGELOG.md

```diff
@@ -45,6 +46,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed import error causing `/timeSeriesSpark` queries to fail.
 - Fixed bug where domsresults no longer worked after successful matchup
 - Fixed certificate error in Dockerfile
+- SDAP-403: Remote timeout fix and HofMoeller bug fix
```

Comment:
I realized I merged https://github.com/apache/incubator-sdap-nexus/pull/203 but CHANGELOG.md wasn't updated so I squeezed that into this PR for completeness.